client/app/bundles/shared/components/Dropdown/DropdownGhost.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import Dropdown from './Dropdown';
+import css from './Dropdown.scss';
+
+export default class DropdownGhost extends Component {
+  render() {
+    return (
+      <div className={css.ghost} >
+        <Dropdown />
```

Comment:
Let's pass a prop to change the styling of the component! See @baohouse's [suggestion here](https://github.com/ifmeorg/ifme/pull/791#issuecomment-362189533).